clonelists/Nintendo - Nintendo 3DS (Digital) (CDN) (No-Intro).json

```diff
@@ -12,13 +12,27 @@
 				{"searchTerm": "Gekiyaba Runner"}
 			]
 		},
+		{
+			"group": "2 Fast 4 Gnomz",
+			"titles": [
+				{"searchTerm": "Blubles", "fake": 3},
+				{"searchTerm": "Gekiyaba Runner"}
+			]
+		},
+		{
+			"group": "SWANG",
+			"titles": [
+				{"searchTerm": "2 Fast 4 Gnomz"},
+				{"searchTerm": "Gekiyaba Runner"}
+			]
+		},
 		{
 			"group": "3D Machine Series, The - Episode 1",
 			"categories": ["Video"],
 			"titles": [
 				{"searchTerm": "3D Machine Series, The - Episode 1"}
 			]
-		},
+		}
 		{

```

Comment:
> ### :gear: Automated review comment

Invalid JSON found on or before this line (36). Fix the error to continue.

There might be more invalid JSON in this file, but only one line can be checked for at a time. To speed up error checking, try an [online JSON validator](https://jsonlint.com/), or use an IDE that can lint JSON like [Visual Studio code](https://code.visualstudio.com/) to find errors before updating your PR.